pkg/addons/data/DaemonSet.apps/openshift-azure-logging/mdsd.yaml

```diff
@@ -46,26 +46,37 @@ spec:
           name: fluentd-config
         - mountPath: /var/log
           name: hostlog
-      - args:
+      - command:
+        - /bin/bash
         - -c
-        - ./bootstrap.sh
+        - |
+          #!/bin/bash
+          sed -e s/_ACCOUNT_/$ACCOUNT/g -e s/_NAMESPACE_/$NAMESPACE/g -e s/_REGION_/$REGION/g -e s/_SUBSCRIPTIONID_/$SUBSCRIPTION_ID/g -e s/_RESOURCEGROUPNAME_/$RESOURCE_GROUP_NAME/g -e s/_RESOURCENAME_/$RESOURCE_NAME/g /etc/mdsd.d/template/mdsd.xml > /etc/mdsd.d/mdsd.xml
```

Comment:
let's get rid of this now rather than waiting more - make a function in config.Derived along the lines of https://gist.github.com/jim-minter/58c203894459498b1c9e295e433b5ad2